poetry/core/packages/utils/utils.py

```diff
@@ -1,6 +1,13 @@
 import os
 import posixpath
 import re
+import sys
+
+from typing import Union
+
+from poetry.core._vendor.six.moves.urllib.parse import unquote  # noqa
```

Comment:
Just out of curiosity, why is there a `#noqa` flag here? Is one of the linters acting funny?

Reply:
Because this uses `._vendor`, linters do not like this.